pkg/status/transitions.go

```diff
@@ -0,0 +1,18 @@
+package status
+
+type Transition struct {
+	Id   string `json:"id"`
+	Name string `json:"name"`
+	To   To     `json:"to"`
+}
+
+type To struct {
+	Name string `json:"name"`
+}
+
+type Transitions struct {
+	Expand      string       `json:"expand"`
+	Transitions []Transition `json:"transitions"`
+}
+
+var TransitionsSlice Transitions
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported var TransitionsSlice should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5024349)